labs.yml

```
@@ -10,6 +10,11 @@ min_python: 3.10
 commands:
   - name: analyze
     description: Analyze source code from existing technology
+    flags:
+      - name: source-directory
+        description: Source Directory to Analyze
+      - name: output-file-name
+        description: Output File Name for Analysis Report, including full path, includes file extension(.xlsx)
```

What do you think about this?
```suggestion
      - name: report-file
        description: Name of (.xlsx) report file to write
```
 - `report`: maybe more intuitive?
 - No `-name`: redundant for a command argument.

---

Done